valheim_plus.cfg

```diff
@@ -368,3 +368,43 @@ wagonBaseMass=20
 
 ; This value changes the game physical weight of Vagons by +/- more/less from item weight inside. The value 50 would increase the weight by 50% more. The value -100 would remove the entire extra weight.
 wagonExtraMassFromItems=0
+
+[Effects]
+
+; Change false to true to enable this section
+enabled=false
+
```

Comment:
what is being enabled here? everything is commented out

Reply:
Its commented, but you can enable buff changes with configuration such as:

```
; Wet Debuff reduction Example
wet.staminaRegenMultiplier = -5
```